package/MDAnalysis/coordinates/base.py

```diff
@@ -216,6 +224,9 @@ def __init__(self, n_atoms, **kwargs):
         self.has_forces = kwargs.get('forces', False)
 
         self._unitcell = self._init_unitcell()
+        
+        # set up aux namespace for adding auxiliary data
+        self.aux = Namespace()
```

Comment:
Yeah, I like it with the Namespace; it doesn't make much difference, but it's that little bit easier to type! (and it's nice to be lazy sometimes >_< )